uber/templates/preregistration/index.html

```diff
@@ -110,10 +110,10 @@
             <br/>If you pay now and cannot pick up your badge, there are no refunds!</strong></div>
         <div class="row">
             <a href="index?payment_method={{ CASH }}"><button class="btn btn-success">Pay with Cash</button></a>
-            {% if not OFFLINE_MODE %}{% stripe_form take_payment charge %}{% endif %}
+            {% if not OFFLINE_MODE %}{% stripe_form prereg_payment charge %}{% endif %}
             <a href="index?payment_method={{ SQUARE }}"><button class="btn btn-info">Pay with Card at the Desk</button></a>
```

Comment:
preregistration.py take_payment() doesn't exist, prereg_payment() does, and it's what master is using, so switched to that.